UnitTests/StreamWithTimeoutDecoratorTest.cs

```diff
@@ -292,57 +292,57 @@ public void WriteByte_IsNotOverridden ()
     [Test]
     public void Write_InnerCannotTimeout ()
     {
-      _innerStreamMock.Expect (s => s.CanTimeout).Return (false);
+      _innerStreamMock.SetupGet (s => s.CanTimeout).Returns (false).Verifiable();
       byte[] buffer = new byte[10];
-      _innerStreamMock.Expect (s => s.Write (buffer, 22, 33));
+      _innerStreamMock.Setup (s => s.Write (buffer, 22, 33)).Verifiable();
 
       _decorator.Write (buffer, 22, 33);
-      _innerStreamMock.VerifyAllExpectations();
+      _innerStreamMock.Verify();
     }
 
     [Test]
     public void Write_UnderlyingStreamTimesOut_ThrowsTimeoutException ()
     {
-      _innerStreamMock.Expect (s => s.CanTimeout).Return (true);
+      _innerStreamMock.SetupGet (s => s.CanTimeout).Returns (true).Verifiable();
       byte[] buffer = new byte[10];
 
-      var waitHandle = MockRepository.GenerateStrictMock<WaitHandle>();
-      waitHandle.Expect (w => w.WaitOne (TimeSpan.FromMilliseconds (123))).Return (false);
+      var waitHandle = new Mock<WaitHandle> (MockBehavior.Strict);
+      waitHandle.Setup (w => w.WaitOne (TimeSpan.FromMilliseconds (123))).Returns (false);
```

Comment:
Missing Verifiable

Reply:
It acts like a stub and is never verified in the old code. I don't think it's needed here.

Reply:
I guess its the same for other occurrences of unverified mocks.

Reply:
But is the bug that it's supposed to be a stub or that it's supposed to be verified?

Reply:
In any case, I would still suggest adding it regardless, since it simplifies reviews

Reply:
We probably just need @MichaelKetting to decide that

Reply:
verify is probably correcter without trying to optimize the intention of the test.